src/main/java/ca/uhn/fhir/jpa/starter/common/validation/RepositoryValidationInterceptorFactoryDstu3.java

```diff
@@ -47,7 +47,7 @@ public RepositoryValidationInterceptorFactoryDstu3(
 	public RepositoryValidatingInterceptor buildUsingStoredStructureDefinitions() {
 
 		IBundleProvider results = structureDefinitionResourceProvider.search(
-				new SearchParameterMap().add(StructureDefinition.SP_KIND, new TokenParam("resource")));
+				new SearchParameterMap().setLoadSynchronousUpTo(Integer.MAX_VALUE).add(StructureDefinition.SP_KIND, new TokenParam("resource")));
```

Comment:
can you not just set `setLoadSynchronous()` if you want to do a full sync search?

Reply:
documentation is a bit unclear what the difference is, but yeah - sure. - I'll adjust it